config/schema/field_encrypt.schema.yml

```diff
@@ -14,6 +14,9 @@ field.storage.*.*.third_party.field_encrypt:
     encryption_profile:
       type: string
       label: 'Encryption profile'
+    cache_exclude:
+      type: boolean
+      label: 'Exclude from cache'
```

Comment:
I think it's broader than just render cache. When caching the entity object itself, unencrypted field data also shouldn't be stored in the (database) cache...